src/css/base.css

```diff
@@ -1,6 +1,12 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 @import 'src/css/variables/colors.css';
 
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
Is there a reason not to put these on the AppContainer?
https://github.com/Microsoft/YamUI/blob/master/src/components/AppContainer/AppContainer.styles.ts#L21

Reply:
i guess just to avoid more global CSS for SP, but this wouldn't affect their page. should i move the rules over to AppContainer?

Reply:
Oh.. I see.  If it's specifically for the modern web client.  Maybe best would be to add the css there?

Reply:
updated